test/com/amazonaws/geo/util/GeoJsonMapperTest.java

```diff
@@ -0,0 +1,117 @@
+package com.amazonaws.geo.util;
+
+
+import java.io.IOException;
+import java.util.Objects;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
You removed com.fasterxml.jackson from the only 2 references in code, then added it here.
Do we still need this and can re remove it from the dependency tree?

Reply:
@jprosenbaum We got rid of the jackson dependency in the new commit. The initial intent was to have backwards compatibility with 1.x version, but we think we don't really need it (the current lambda backend can still use version 1 of this library).